src/utils.tsx

```diff
@@ -87,3 +87,19 @@ export const formatDateTimeStrings = (
 
   return formattedDate;
 };
+
+/* returns a string with the total rows and current rows for table views */
```

Comment:
MRT should have in built function that does this, use that instead of this

Reply:
I can't find anything that does this. I used what was done in MRTs pagination on [github ](https://github.com/KevinVandy/material-react-table/blob/v2/packages/material-react-table/src/components/toolbar/MRT_TablePagination.tsx) and they don't use a function it is hardcoded like here

Reply:
The total rows should be calculated using the filters no the pagination

Reply:
![image](https://github.com/ral-facilities/inventory-management-system/assets/144466888/1d7c0869-a0f9-4de4-b280-03bf271e3b56)


if you apply filters it still calculates the correct number of rows.

Reply:
you shouldn't need to create a custom function for this